sound/soc/intel/boards/Kconfig

```diff
@@ -97,6 +97,7 @@ config SND_SOC_INTEL_BYTCR_RT5640_MACH
 	tristate "Baytrail and Baytrail-CR with RT5640 codec"
 	depends on I2C && ACPI
 	depends on X86_INTEL_LPSS || COMPILE_TEST
+	depends on GPIOLIB || COMPILE_TEST
```

Comment:
If GPIOLIB is _needed_ for these machine drivers, then should we select it instead?

Reply:
if we use select, we would have a risk of creating a depend-select issue and more randconfig problems. I think it's the responsibility of the user/distro to select dependencies. In practice GPIOLIB is always already selected already so that's not a real issue.